pages/common/cppclean.md

```diff
@@ -0,0 +1,19 @@
+# cppclean
+
+> Find unused code in C++ projects.
+
+- Run in a project's folder:
+
+`cppclean {{path/to/project}}`
+
+- Run on a project where the headers are in the "inc1/" and "inc2/" folders:
+
+`cppclean {{path/to/project}} --include-path={{inc1}} --include-path={{inc2}}`
+
+- Run on a specific file "main.cpp":
+
+`cppclean {{main.cpp}}`
+
+- Run on the current directory, excluding the "node_modules" directory:
```

Comment:
I don't think you will have a "node_modules" directory in a c++ project. Maybe you can change it to "build" ?

Reply:
Eh, I actually took the first folder that I found (which might not have been the smartest thing), and it was a `node_modules` folder. I've been working on some C++ native modules, using the `NaN`/`libuv` libraries, which are installed through npm, and contain tons of C++ that trigger `cppclean`.